bindings/python/release.sh

```diff
@@ -38,16 +38,20 @@ function get_libmongocrypt() {
 }
 
 function build_wheel() {
+    python -m pip install unasync
     python -m pip install --upgrade pip build
     python -m build --wheel
     rm -rf build libmongocrypt pymongocrypt/*.so pymongocrypt/*.dll pymongocrypt/*.dylib
+    python synchro.py
```

Comment:
This seems wrong. Aren't we suppose to run synchro _before_ committing the code? So there should never be a need to run it here.